instrumentation/finagle-http-23.11/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/finaglehttp/v23_11/FinagleHttpInstrumentationModule.java

```diff
@@ -27,9 +29,16 @@ public List<TypeInstrumentation> typeInstrumentations() {
   }
 
   @Override
-  public boolean isIndyModule() {
-    // injects helpers to access package-private members
-    return false;
+  public String getModuleGroup() {
+    // relies on netty and needs access to common netty instrumentation classes
+    return "netty";
+  }
+
+  @Override
+  public List<String> injectedClassNames() {
+    return Arrays.asList(
```

Comment:
(did I get this right?)
```suggestion
    // these are injected so that they can access package-private members
    return Arrays.asList(
```

Reply:
That's correct.